not-to-release/sources/answers/20111106215236AAycANO_ans.xml.conllu

```diff
@@ -333,7 +333,7 @@
 # sent_id = answers-20111106215236AAycANO_ans-0024
 # text = Monday thru Friday
 1	Monday	Monday	PROPN	NNP	Number=Sing	0	root	0:root	_
-2	thru	thru	ADP	IN	_	3	case	3:case	_
+2	thru	through	ADP	IN	Abbr=Yes	3	case	3:case	_
```

Comment:
I agree that we should use the canonical spelling for the lemma but does this really count as an abbreviation?

Reply:
Okay, now that I looked at the other examples, it probably makes sense to treat this as an abbreviation that is used in texting etc.